docs/hub/academia-hub.md

```diff
@@ -3,61 +3,51 @@
+3. **Prototype** ideas using interactive Spaces and hosted notebooks.
+4. **Train & scale** with managed GPUs and tracked runs.
+5. **Publish & share** using model cards, DOIs, and dataset releases.
+6. **Preserve** your work for reproducibility and future research. All backed by enterprise-grade infrastructure, institutional governance, and storage that grows with your needs.
 
-1. Check your eligibility (see below)
-2. Ask your university’s IT or Procurement Team to <a href="https://huggingface.co/contact/sales?from=academia" target="_blank">get in touch</a> to initiate the subscription process to the Academia Hub today. 
-Academia Hub can not be initiated by students themselves. 
-3. When Academia Hub is enabled, any affiliated user will need to add their `@your-university-name.edu` email address (or other university domain) to their HF account.
+## Key features of Academia Hub
 
-#### Eligibility
+***For researchers and students***
+- **Storage**: 1 TB private storage per seat (e.g., 400 seats = 400 TB) powered by Xet, purpose-built for versioning large AI models and datasets; expanded public storage; Dataset Viewer for private datasets
+- **Hosting & demos**: Spaces Hosting for scalable AI demos and applications powered by ZeroGPU (5× priority quota); Dev Mode with SSH/VS Code access for development
+- **Compute**: Priority GPU access (H100/H200) for training; managed runs with experiment tracking
+- **Collaboration**: Team workspaces with version control, peer review, and shared governance
+- **Publishing**: Share your work with the global AI research community through citable releases with model cards, dataset cards, and DOIs
 
-**Who is Academia Hub for?**
-- **Students:** Unlock powerful features to learn about AI and Machine learning in the most efficient way.
-- **Researchers:** Collaborate with peers using the standard AI ecosystem of tools.
-- **Faculty members:** Enhance your classes' projects with PRO capabilities.
+***For administrators***
+- **Pricing**: $10/seat/month (volume-based pricing); $2/seat/month compute credits included (top-ups available)
+- **Admin & security**: University-level seat management for researchers, professors, and students; centralized administration; SSO with university domain
 
-**Requirements**
-- Must possess a valid university or college email address.
-- Open to all students regardless of discipline or level of study.
-- Pricing: Academia Hub is priced based on volume of usage and number of active users at your institution.
+***For your research community***
+- **Community & resources**: Connect with peers; curated models/datasets/projects for academia
 
+## How to get started
 
+1. Ask your university's IT or Procurement Team to <a href="https://huggingface.co/contact/sales?from=academia" target="_blank">get in touch</a> to initiate the subscription process to the Academia Hub today.
+Academia Hub cannot be initiated by researchers or students themselves. 
```

Comment:
Does it still make sense to capture interest from researchers/students somewhere?